client/my-sites/earn/components/form-text-input-with-value-generation/test/index.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { render, screen } from '@testing-library/react';
+import FormTextInputWithValueGeneration from '../index';
+
+describe( 'FormTextInputWithValueGeneration', () => {
+	test( 'should render', () => {
+		const { container } = render( <FormTextInputWithValueGeneration /> );
+		expect( container ).toMatchSnapshot();
```

Comment:
Can we refactor the snapshot to something more meaningful, or remove it if there's no meaningful thing to test?